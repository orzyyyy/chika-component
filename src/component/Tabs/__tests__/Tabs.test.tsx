import React from 'react';
import { mount } from 'enzyme';
import 'nino-cli/scripts/setup';
let Tabs;
switch (process.env.LIB_DIR) {
  case 'lib':
    Tabs = require('../../../../lib/component/Tabs').default;
    break;
  case 'dist':
    Tabs = require('../../../../dist/lib/Tabs').default;
    break;
  default:
    Tabs = require('..').default;
    break;
}

describe('Tabs', () => {
  it('render correctly', () => {
    const wrapper = mount(<Tabs />);
    expect(wrapper).toMatchSnapshot();
  });
});
