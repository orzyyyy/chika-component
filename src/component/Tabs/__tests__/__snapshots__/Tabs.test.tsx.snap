// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs render correctly 1`] = `
<Tabs>
  <div
    className="Tabs"
  >
    <ul
      className="container"
      key="container_ul"
    />
    <div
      className="underline-container"
      key="underline"
    >
      <div
        className="underline"
        style={
          Object {
            "transform": "translate3d(0px, 0, 0)",
            "width": 0,
          }
        }
      />
    </div>
  </div>
</Tabs>
`;
